src/components/AccountBalance.tsx

```diff
@@ -10,7 +11,8 @@ const AccountBalance: FC<{
 }> = ({
   accountId,
   skeletonProps,
-  renderBalance = balance => balance?.confirmed?.toString() || 0,
+  renderBalance = balance =>
+    CurrencyUtils.renderIron(balance?.confirmed || '0'),
```

Comment:
May be better to use `encodeIron` function (will return 0.1)? `renderIron` will return 8 digits after dot (ex 0.10000000).

Reply:
As I got from conversation it need 8 decimal places, @lwisne is it correct?